caliban_toolbox/dataset_builder.py

```diff
@@ -534,7 +592,7 @@ def build_dataset(self, tissues='all', platforms='all', output_shape=(512, 512),
         Raises:
             ValueError: If invalid resize parameter supplied
         """
-        if self.all_tissues == []:
+        if self.all_tissues is None:
```

Comment:
You could generalize this to `if not self.all_tissues` which would work for `None` and empty lists/arrays.

Reply:
Just tried this, the problem is that when it's not an empty array, leads to the following error:
`ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
`